docker-images/dockerfiles/model-compiler.dockerfile

```diff
@@ -33,18 +33,19 @@ ARG TENSORRT_VERSION
 ARG CUDA_VERSION
 ENV TENSORRT_VERSION=${TENSORRT_VERSION}
 ENV CUDA_VERSION=${CUDA_VERSION}
+ENV PIP_FIND_LINKS='https://release.oneflow.info/'
 
 RUN . /etc/os-release && \
-    apt-get update && \
-    apt-get install --no-install-recommends -y wget && \
-    wget "https://developer.download.nvidia.com/compute/cuda/repos/$ID$(echo $VERSION_ID | tr -d .)/x86_64/7fa2af80.pub" -O /etc/apt/trusted.gpg.d/cuda.asc && \
-    apt-get autoremove --purge -y wget && \
-    apt-get clean && \
-    find /var/lib/apt/lists -delete
-
-RUN . /etc/os-release && \
-    echo "deb https://developer.download.nvidia.com/compute/cuda/repos/$ID$(echo $VERSION_ID | tr -d .)/x86_64 /\n\
-    deb https://developer.download.nvidia.com/compute/machine-learning/repos/$ID$(echo $VERSION_ID | tr -d .)/x86_64 /" >> /etc/apt/sources.list
+	apt-get update && \
+	apt-get install --no-install-recommends -y \
+		gnupg curl software-properties-common && \
```

Comment:
Better to merge these two lines:
`
	apt-get install --no-install-recommends -y \
		gnupg curl software-properties-common && \
`
into one line.